source/renderer/app/components/wallet/transactions/WalletTransactionsList.js

```diff
@@ -248,14 +251,16 @@ export default class WalletTransactionsList extends Component<Props> {
         />
       ) : null;
 
-    const syncingTransactionsSpinner = isRestoreActive ? (
-      <div className={styles.syncingTransactionsWrapper}>
-        <LoadingSpinner big />
-        <p className={styles.syncingTransactionsText}>
-          {intl.formatMessage(messages.syncingTransactionsMessage)}
-        </p>
-      </div>
-    ) : null;
+    // @TX TODO
```

Comment:
@daniloprates what's with this "TODO"? Don't we need a spinner here while restoring?